src/MakersOfDenmark.Domain/Models/User/MODRole.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MakersOfDenmark.Domain.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MakersOfDenmark.Domain.Models.User
+{
+    public class MODRole : Entity<int>
```

Comment:
Vi har MODRole og MakerSpaceRole. Kan det skabe forvirring?

Reply:
Jeg synes vi skal kalde MODRole for UserRole.

Reply:
<img width="548" alt="User" src="https://user-images.githubusercontent.com/32497085/98556474-60901100-22a3-11eb-890d-d36e679f87d0.png">
Efter at have tygget lidt mere på den, så synes jeg faktisk vi skal gøre følgende.

Vi har vores bruger som hedder MODUser. Den rolle, som denne får tildelt, bør derfor hedde MODUserRole. Endeligt, så kan MakerSpaceRole blive ved med at hedde det, da dette er de generiske/enum-roller man kan være på et MakerSpace. Lad mig høre hvad I tænker